app/views/layouts/_lte_sidebar.html.erb

```diff
@@ -59,14 +59,13 @@
           </li>
         </ul>
       </li>
-      <li>
       <li class="<%= 'active' if current_page?(requests_path) %>">
         <%= link_to(requests_path) do %>
           <i class="fa fa-file-text"></i> <span>Requests</span>
         <% end %>
       </li>
-      <li class="<%= active_class(['distributions']) %>">
-        <%= link_to(distributions_path(organization_id: current_user.organization)) do %>
+      <li class="<%= 'active' if current_page?(distributions_path) %>">
```

Comment:
I realize we have this format for `requests` above, but I *think* it might be better to stick with the helper here.

Reply:
The problem here is that "pick-ups" was enabling "distributions", so when I went to the "pick-ups" page, it confused me about which page was really active, I think it would look better this way, but if you think it's best to leave it the way it is, no problem.

Reply:
ahhhhhh!! I get it.

`#pickups` is an action in the `Distributions` controller, and this method assumes that each section is governed by a single controller. OK  -- I'm good with the change you made here :+1: We'll need to find a better way to address this, but I think that's out-of-scope for this PR so don't worry about it :) 

Could you make the `distributions_path` incorporate the `organization_id:` again, like it did previously? I don't think it's absolutely necessary (I'm pretty sure it's inferred) but the resource requires the organization for context. Thank you! :bowing_man: